<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalProgress</class>
 <widget class="QWidget" name="CalProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Printing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>11</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="currentLabel">
     <property name="text">
      <string>Current Page</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Digikam::DProgressWdg" name="currentProgress"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="totalLabel">
     <property name="text">
      <string>Total Pages</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Digikam::DProgressWdg" name="totalProgress"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="finishLabel"/>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Digikam::DProgressWdg</class>
   <extends>QProgressBar</extends>
   <header>dprogresswdg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
